osu.Game/Overlays/Profile/UserGraph.cs

```diff
@@ -292,4 +294,12 @@ protected override void PopIn()
             protected override void PopOut() => this.FadeOut(200, Easing.OutQuint);
         }
     }
+
+    public class UserGraphTooltipContent
+    {
+        // todo: change to init-only on C# 9
```

Comment:
If you really need an immutable structure here then the lack of init setters isn't really an excuse, is it? Constructors still exist.